metrics/density/footprint_data.md

```diff
@@ -17,21 +17,20 @@ Some variables affect how the payload is executed. The `RUNTIME` and `PAYLOAD`
 arguments directly affect the payload execution with the following line in
 the script:
 
-`$ docker run -tid --runtime=$RUNTIME $PAYLOAD_RUNTIME_ARGS $PAYLOAD $PAYLOAD_ARGS`
+`$ ctr run --memory-limit $PAYLOAD_RUNTIME_ARGS --rm --runtime=$CONTAINERD_RUNTIME $PAYLOAD $NAME sh -c $PAYLOAD_ARGS`
 
 Other settings affect how memory footprint is measured and the test termination
 conditions.
 
 | Variable | Function
 | -------- | --------
-| `PAYLOAD` | The docker image to run
-| `PAYLOAD_ARGS` | Any arguments passed into the docker image
-| `PAYLOAD_RUNTIME_ARGS` | Any extra arguments passed into the docker `run` command
+| `PAYLOAD` | The ctr image to run
+| `PAYLOAD_ARGS` | Any arguments passed into the ctr image
+| `PAYLOAD_RUNTIME_ARGS` | Any extra arguments passed into the ctr `run` command
```

Comment:
Could you put `ctr` in backticks everywhere in this table (making this cell `ctr run`).

Reply:
@jodh-intel changes applied